keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/config.h

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* #define DEBUG_LED_PIN      GP25 */
+#define CRC8_USE_TABLE
+#define CRC8_OPTIMIZE_SPEED
+
+/* Reset. */
+#define RESET_PIN GP22
+#define RESET_PIN_DOUBLE_TAP_RESET_TIMEOUT 1000U
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+/* split things */
+#define SPLIT_POINTING_ENABLE
+#define SPLIT_HAND_PIN      GP28  // high = left, low = right
+
+/* Key matrix configuration. */
+#define MATRIX_ROW_PINS \
+    { GP4, GP5, GP6, GP7, GP8, GP9 }
+#define MATRIX_COL_PINS \
+    { GP10, GP11, GP12, GP13, GP14, GP15 }
+
+/* WS2812 RGB LED strip input and number of LEDs */
+#define WS2812_PIO_USE_PIO1
+#define RGB_DI_PIN          GP21
```

Comment:
by the time this gets merged...
```suggestion
#define WS2812_DI_PIN       GP21
```